dropwizard-app/pom.xml

```diff
@@ -42,9 +42,30 @@
         </dependency>
 
         <dependency>
+            <groupId>com.smoketurner</groupId>
+            <artifactId>dropwizard-swagger</artifactId>
+            <version>1.0.5-3</version>
```

Comment:
I started up the server using `java -jar dropwizard-app/target/dropwizard-app-0.1-SNAPSHOT.jar server dropwizard-app/src/test/resources/config/test.yml`, but when I request http://localhost:8080/swagger.json, I get an HTTP 500 and the app prints a stack trace pointing to https://github.com/swagger-api/swagger-core/blob/master/modules/swagger-core/src/main/java/io/swagger/core/filter/SpecFilter.java#L47, which suggests something stopped swagger obtaining resource paths.